src/Form/FieldCheckboxMultiple.js

```diff
@@ -18,8 +18,7 @@ class FieldCheckboxMultiple extends Util.mixin(BindMixin) {
       props.name,
       {
         name: name,
-        checked: modelState.checked,
-        indeterminate: modelState.indeterminate
+        checked: modelState
```

Comment:
nope, indeterminate is still a property on the definition. it's just that indeterminate will come from the parent, the checkbox will never hold the value of indeterminate in state